mlflow/utils/docstring_utils.py

```diff
@@ -117,6 +117,29 @@ def decorator(func):
 # `{{ ... }}` represents a placeholder.
 LOG_MODEL_PARAM_DOCS = ParamDocs(
     {
+        "conda_env": """
+Either a dictionary representation of a Conda environment or the path to a conda environment yaml
+file. If provided, this describes the environment this model should be run in. At minimum, it
+should specify the dependencies contained in :func:`get_default_conda_env()`. If ``None``, a conda
+environment with pip requirements inferred by :func:`mlflow.infer_pip_requirements` is added to
+the model. If the auto requirements inference fails, it falls back to using
```

Comment:
Awesome! Can we add this fallback information to the `pip_requirements` param doc section as well?